lib/clAdapters/googleAdapter.js

```diff
@@ -1,37 +1,386 @@
 'use strict';
 
-var util = require('util'),
-  BaseAdapter = require('./baseAdapter');
-
-/**
- * GoogleAdapter
- *
- * `credentials` format:
- * ```
- * {
- *   email: 'test@test.com',
- *   password: 'password',
- *   account: 123456,
- *   role: 3
- * }
- * ```
- * @class
- * @return {GoogleAdapter}
- */
+var uuid = require('node-uuid');
+var crypto = require('crypto');
+var rp = require('request-promise');
+var util = require('util');
+var BaseAdapter = require('./baseAdapter');
+var GoogleMail = require('./google-js.js');
+var moment = require('moment');
+var querystring = require('querystring');
+var _ = require('lodash');
+require('es6-promise').polyfill();
```

Comment:
Minor: The call to `polyfill` from the original `lib/index.js` file polyfills globally so don't need to call this in multiple places.